modules/AOS_PDF_Templates/templateParser.php

```diff
@@ -101,10 +101,18 @@ function parse_template_bean($string, $key, &$focus)
                 $value = rtrim(rtrim(format_number($value), '0'), $sep[1]) . $app_strings['LBL_PERCENTAGE_SYMBOL'];
             }
             if (strpos($name, 'date') > 0 || strpos($name, 'expiration') > 0) {
-                if ($value != '') {
-                    $dt = explode(' ', $value);
-                    $value = $dt[0];
-                }
+			
+                if(strpos($name, 'dates') > 0)
+				{
```

Comment:
Be aware of the bracket placement for control structures - See our updated coding standards - [https://suitecrm.com/wiki/index.php/Coding_Standards#Brace_Style]